ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fragmentcontainer/v1/fragmentcontainer/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2022 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:granite="http://www.adobe.com/jcr/granite/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+  jcr:primaryType="nt:unstructured"
+  jcr:title="Adaptive Form Fragment Container"
+  sling:resourceType="cq/gui/components/authoring/dialog"
+  extraClientlibs="[core.forms.components.container.v1.editor]"
+  helpPath="https://www.adobe.com/go/aem_af_cmp_fragmentcontainer_v1"
+  trackingFeature="core-components:adaptiveform-fragmentcontainer:v1">
+    <content
+      jcr:primaryType="nt:unstructured"
+      granite:class="cmp-adaptiveform-container__editdialog cmp-adaptiveform-container__editdialog--fragmentcontainer"
```

Comment:
Can you fix the BEM, ideally it should be this, cmp-adaptiveform-fragmentcontainer__editdialog ? You have misused the modifier in BEM, fragmentcontainer is not a modifier, its an element ?

Reply:
ok